helm/resource_release.go

```diff
@@ -302,6 +302,9 @@ func resourceRelease() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "Add a custom description",
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
I have suppressed this because when you import a release and don't set a description it will show up in the diff as being changed from `Install complete` to null. I have made the assumption here that if the user hasn't set a description field then they probably don't care to capture the default one that gets set by helm.

Reply:
Would setting this to `Computed` also have worked? That's sort of the canonical way to treat such cases.
Regardless, I'm with you that Description is likely not useful in any non-human context so not capturing in state should not make much of a difference.

Reply:
That's what I thought initially, but they added the `--description` feature to the helm CLI so there are potentially situations where the operator will want to explicitly set this field. See #439